frontend/src/components/AccountDetailsEditContainer.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <div>
+    <ContainerComponent>
+      <template #Header>
+        <h1 class="underline text-xl">Daten ändern:</h1>
+      </template>
+
+      <template #default>
+        <div class="text-gray-400 p-1 text-xs">
+          Hinweis: Sie müssen nicht alle Felder ausfüllen, leere Felder werden
+          nicht geändert
+        </div>
+
+        <table class="w-full">
+          <tr>
+            <td class="p-1">Nutzername:</td>
+            <td class="p-1 w-2/3">
+              <TextInputComponent
+                :disabled="disableInputShowSpinner || deleted"
+                place-holder="Nutzername"
+                :startText="originalUser.name"
+                @input-changed="(val) => (nameField = val)"
+                :max-length="128"
+              />
+            </td>
+          </tr>
+          <tr>
+            <td class="p-1">E-mail:</td>
+            <td class="p-1">
+              <TextInputComponent
+                :disabled="disableInputShowSpinner || deleted"
+                place-holder="E-mail"
+                :startText="originalUser.mail"
+                @input-changed="(val) => (mailField = val)"
+                :max-length="320"
+              />
+            </td>
+          </tr>
+          <tr>
+            <td class="p-1">Passwort:</td>
+            <td class="p-1">
+              <TextInputComponent
+                :disabled="disableInputShowSpinner || deleted"
+                :hidden="true"
+                place-holder="Passwort"
+                @input-changed="(val) => (pwField = val)"
+                :max-length="128"
+              />
+            </td>
+          </tr>
+          <tr>
+            <td class="p-1">Passwort wiederholen:</td>
+            <td class="p-1">
+              <TextInputComponent
+                :disabled="disableInputShowSpinner || deleted"
+                :hidden="true"
+                place-holder="Passwort wiederholen"
+                @input-changed="(val) => (pwFieldControl = val)"
+                :max-length="128"
+              />
+            </td>
+          </tr>
+          <tr :class="showadminview ? '' : 'collapse'">
+            <td></td>
+            <td>
+              <div>
+                <input
+                  :disabled="disableInputShowSpinner || deleted"
+                  type="checkbox"
+                  class="cursor-pointer m-1"
+                  v-model="isAdmin"
+                  :checked="isAdmin"
+                />
+                <label class="cursor-pointer" @click="toggleCheckBox">
```

Comment:
ambigous name with so many inputs

Reply:
we only have one checkbox

Reply:
I know, what I ment is in case more get added for some reason. And even now it is a very undescribtive name